<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageTab</class>
 <widget class="QWidget" name="ImageTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="ImagePlotWidget" name="widget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblX">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>x: --</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblY">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>y: --</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="sizeWidget" native="true"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbComplex">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>26</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Amplitude</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Phase</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Real</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Imaginary</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImagePlotWidget</class>
   <extends>QWidget</extends>
   <header>controls/imageplot/imageplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
